components/automate-ui/src/app/modules/infra-proxy/edit-infra-node-modal/edit-infra-node-modal.component.scss

```diff
@@ -0,0 +1,236 @@
+        left: 398px;
+        right: 398px;
+        top: 462px;
+        border: 1px solid $chef-form-input;
+        box-sizing: border-box;
+        border-radius: 4px;
+        margin-top: 10px;
+      }
+
+      #default, #override {
+        height: 302px;
+        max-height: 302px;
+        min-height: 302px;
+        max-width: 100%;
+        min-width: 100%;
+      }
+
+      ::ng-deep #default::-webkit-scrollbar {
+        width: 5px;
+        background: $chef-form-input;
+        border-radius: 5px;
+      }
+
+      ::ng-deep #default::-webkit-scrollbar-thumb {
+        background: $chef-modal;
+        border-radius: 5px;
+      }
+
+      ::ng-deep #override::-webkit-scrollbar {
+        width: 5px;
+        background: $chef-form-input;
+        border-radius: 5px;
+      }
+
+      ::ng-deep #override::-webkit-scrollbar-thumb {
+        background: $chef-modal;
+        border-radius: 5px;
+      }
+    }
+
+    .button-bar-box {
+      text-align: right;
+      margin-top: 20px;
+
+      chef-loading-spinner {
+        position: relative;
+        top: 2px;
+        margin-right: 5px;
+      }
+    }
+
+    .button-bar-constraint {
+      text-align: right;
+      margin-top: 100px;
+
+      chef-loading-spinner {
+        position: relative;
+        top: 2px;
+        margin-right: 5px;
+      }
+    }
+  }
+
+  ::ng-deep {
+    .modal {
+      overflow-x: hidden;
+      padding: 32px;
+      outline: none;
+      box-shadow: 0px 0px 30px 0px $chef-modal;
+      width: 708px;
+    }
+
+    .modal::-webkit-scrollbar {
+      width: 8px;
+      background-color: $chef-modal-scrollbar;
+    }
+
+    .modal::-webkit-scrollbar-thumb {
+      border-radius: 10px;
+      background-color: $chef-modal;
+    }
+
+    .label {
+      font-size: 16px;
+      font-weight: 400;
+      padding-bottom: 0px;
+    }
+  }
+}
+
+.constraint {
+  width: 61px;
+  height: 20px;
+  left: 398px;
+  top: 278px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0.5px;
+  text-transform: capitalize;
+  margin-top: 50px;
+}
+
+.edit-label {
+  border-bottom: 0.5px solid $chef-key-tab;
+  line-height: 3;
+  margin-bottom: 31px;
+}
+
+.small:first-of-type {
+  margin-top: 10px;
+}
+
+chef-modal ::ng-deep .modal-overlay {
+  background-color: $chef-white;
+  mix-blend-mode: normal;
+  opacity: 0.7;
+}
+
+.pages-body {
+  padding-top: 0;
+
+  ::ng-deep button.nav-tab.active {
+    background: $chef-primary-bright;
+  }
+
+  ::ng-deep button.nav-tab:focus {
+    outline: none;
+  }
+
+  ::ng-deep .nav-tab:before {
+    height: 4px;
+    background: $chef-tab-hover;
+  }
+}
+
+.button-bar-box {
+  chef-button {
+    height: 36px;
+  }
+
+  chef-button#create-default-button-object-modal {
+    margin: 10px 0px 0px 16px;
+  }
+
+  chef-button#create-override-button-object-modal {
+    margin: 10px 0px 0px 16px;
+  }
+
+  chef-button#create-runlist-button-object-modal {
+    margin: 10px 0px 0px 16px;
+  }
```

Comment:
[nit] - since these are all the same you could stack them or rename.... up to you

  chef-button#create-default-button-object-modal,
chef-button#create-override-button-object-modal,
chef-button#create-runlist-button-object-modal { 
margin: 10px 0px 0px 16px;
}